WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController.swift

```diff
@@ -675,7 +675,34 @@ extension GutenbergViewController: GutenbergBridgeDelegate {
             return
         }
 
-        storyEditor = StoryEditor.editor(post: post, mediaFiles: files, publishOnCompletion: false, updated: { [weak self] result in
+        do {
+            try showEditor(files: files)
+        } catch let error {
+            switch error {
+            case StoryEditor.EditorCreationError.unsupportedDevice:
+                let title = NSLocalizedString("Unsupported Device", comment: "Title for stories unsupported device error.")
+                let message = NSLocalizedString("We are still working on the Stories editor for iPad. In the meantime, please try Stories on your iPhone.", comment: "Message for stories unsupported device error.")
```

Comment:
I might be a little overly cautious about the wording here, but I'm wondering if it could run afoul of Apple's "no betas" rule if we get a reviewer that's particularly fussy.  Instead of "We are still working on the Stories editor for iPad" would it be safer to say something like "The Stories editor for is not currently available for your iPad"?